truss/contexts/local_loader/truss_file_watcher.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+from threading import Thread
+
+from truss.util.path import copy_file_path
+from watchfiles import Change, watch
+
+
+class TrussFilesWatcher(Thread):
```

Comment:
This seems more like a file syncer, since it hard codes the opertion to perform when something changes. May be name it so. Or if the intention is for it to be generic then it should perhaps take an observer as input, to which it informs about the changes.